src/Files.App/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -428,6 +423,29 @@ private async void FileList_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
 				FileList.SelectedIndex = 0;
 				e.Handled = true;
 			}
+			else if (e.KeyStatus.IsMenuKeyDown)
+			{
+				if (shiftPressed && e.Key == VirtualKey.Add)
+				{
+					// Unfocus the ListView so keyboard shortcut can be handled (alt + shift + "+")
+					this.Focus(FocusState.Pointer);
+					return;
+				}
+
+				// Unfocus the GridView so keyboard shortcut can be handled
+				switch (e.Key)
+				{
+					case VirtualKey.Up:
+						NavToolbar?.Up.Focus(FocusState.Pointer);
```

Comment:
Any reason GridView is handled differently wrt to Details & Column? (`NavToolbar?.Up.Focus()` VS `this.Focus()`)

Reply:
Using `this.Focus()` in Columns makes the behaviour of shortcuts unpredictable. Sometimes the focus remains on the `GridView` and this element handles arrows (resulting in focus moving between cells). I can't reproduce this issue in Details.